src/operator/nn/fully_connected.cc

```diff
@@ -210,11 +222,11 @@ inline static bool FCStorageType(const nnvm::NodeAttrs& attrs,
   return dispatched;
 }
 
-inline static bool BackwardFCStorageType(const nnvm::NodeAttrs& attrs,
-                                         const int dev_mask,
-                                         DispatchMode* dispatch_mode,
-                                         std::vector<int> *in_attrs,
-                                         std::vector<int> *out_attrs) {
+static bool BackwardFCStorageType(const nnvm::NodeAttrs& attrs,
```

Comment:
Why remove inline?

Reply:
I have explained this before countless times. This is just code bloat, the function has several calls and a loop inside, inline does nothing good here. The compiler knows when to inline.

Reply:
Please add reference. According to this article, the compiler can automatically detect whether inline is needed. https://www.geeksforgeeks.org/inline-functions-cpp/

Moreover, if this does not affect the feature you are developing in this PR, please not change.